src/main/java/ca/fxco/pistonlib/impl/BlockEntityPostLoad.java

```diff
@@ -0,0 +1,16 @@
+package ca.fxco.pistonlib.impl;
+
+/**
+ * If a block entity implements this interface. They will be run once, before there first tick
+ */
+public interface BlockEntityPostLoad {
+
+    boolean shouldPostLoad();
+
+    /**
+     * This only gets first once, before your first tick!
```

Comment:
should the first 'first' be 'fired' or is my midnight brain bad at English

Reply:
ah good spot xD